docs/architecture/basic-normalization.md

```diff
@@ -0,0 +1,162 @@
+# Basic Normalization
+
+## Overview
+
+Basic Normalization uses a fixed set of rules to map a json object from a source to the types and format that are native to the destination. For example if a source emits data that looks like this:
+```json
+{
+  "make": "alfa romeo",
+  "model": "4C coupe",
+  "horsepower": "247"
+}
+```
+Then basic normalization would create the following table:
+
+```sql
+CREATE TABLE "cars" (
+    -- metadata added by airbyte
+    "_cars_hashid" VARCHAR, -- uuid assigned by airbyte derived from a hash of the data.
+    "emitted_at" TIMESTAMP_WITH_TIMEZONE, -- time at which the record was emitted.
+    "normalized_at" TIMESTAMP_WITH_TIMEZONE, -- time at which the record was normalized.
+
+    -- data
+    "make" VARCHAR,
+    "model" VARCHAR,
+    "horsepower" INTEGER
+);
+```
+
+You'll notice that we add some metadata to keep track of important information about each record.
+
+The [normalization rules](#Rules) are _not_ configurable. They are designed to pick a reasonable set of defaults to hit the 80/20 rule of data normalization. We respect that normalization is a detail-oriented problem and that with a fixed set of rules, we cannot normalize your data in such a way that covers all use cases. If this feature does not meet your normalization needs, we always put the full json blob in destination as well, so that you can parse that object however best meets your use case. We will be adding more advanced normalization functionality shortly.
```

Comment:
I would mentioned something about the fact that we are EL and that DBT is great solution to do the T part